infra/stack.json

```diff
@@ -0,0 +1,415 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "",
+    "Parameters": {
+        "DBUsername": {
+            "NoEcho": "true",
+            "Description": "Username for database access",
+            "Type": "String",
+            "MinLength": "1",
+            "MaxLength": "16",
+            "AllowedPattern": "[a-zA-Z][a-zA-Z0-9]*",
+            "ConstraintDescription": "must begin with a letter and contain only alphanumeric characters."
+        },
+        "DBPassword": {
+            "NoEcho": "true",
+            "Description": "Password database access",
+            "Type": "String",
+            "MinLength": "8",
+            "MaxLength": "41"
+        },
+        "SecretKey": {
+            "NoEcho": "true",
+            "Description": "Secret key for Play application",
+            "Type": "String",
+            "MinLength": "12",
+            "MaxLength": "64"
+        }
+    },
+    "Resources": {
+        "vpc": {
+           "Type" : "AWS::EC2::VPC",
+           "Properties" : {
+              "CidrBlock" : "10.192.0.0/16"
```

Comment:
**Suggestion** (non-blocking): Do you think we shall use a larger mask? e.g. `19`.

Reply:
Thanks - that's a good idea - I'm going to take you up on the "non-blocking" part and ask to make it a change for the future, though.  Everything I have seen so far has used `/16`, and eventually we'll have to peer with the seattle VPC where we might need to change things.